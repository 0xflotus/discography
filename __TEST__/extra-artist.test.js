import testSnapshot from './fixtures/testSnapshot';

test('Should return all fields on extra artist type', async () => {
  const query = `
   query {
     lookup {
        release (id: 2) {
          id
          extraArtists {
            edges {
              node {
                id
                name
                realName
                aliases
                urls
                role
              }
            }
          }
        }
      }
    }
  `;

  await testSnapshot(query);
});
