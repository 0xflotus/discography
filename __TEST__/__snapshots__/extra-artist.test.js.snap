// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should return all fields on extra artist type 1`] = `
Object {
  "data": Object {
    "lookup": Object {
      "release": Object {
        "extraArtists": Object {
          "edges": Array [
            Object {
              "node": Object {
                "aliases": Array [
                  "A.D.1010",
                  "ADNY",
                  "Bob Brewthbaker",
                  "G.O.L.",
                  "Leiva",
                ],
                "id": 26,
                "name": "Alexi Delano",
                "realName": "Alexi Delano",
                "role": "Producer",
                "urls": Array [
                  "http://www.myspace.com/alexidelano",
                  "http://twitter.com/AlexiDelano",
                ],
              },
            },
            Object {
              "node": Object {
                "aliases": Array [
                  "A.D.1010",
                  "ADNY",
                  "Bob Brewthbaker",
                  "G.O.L.",
                  "Leiva",
                ],
                "id": 26,
                "name": "Alexi Delano",
                "realName": "Alexi Delano",
                "role": "Recorded By",
                "urls": Array [
                  "http://www.myspace.com/alexidelano",
                  "http://twitter.com/AlexiDelano",
                ],
              },
            },
            Object {
              "node": Object {
                "aliases": Array [
                  "A.D.1010",
                  "ADNY",
                  "Bob Brewthbaker",
                  "G.O.L.",
                  "Leiva",
                ],
                "id": 26,
                "name": "Alexi Delano",
                "realName": "Alexi Delano",
                "role": "Written-By",
                "urls": Array [
                  "http://www.myspace.com/alexidelano",
                  "http://twitter.com/AlexiDelano",
                ],
              },
            },
            Object {
              "node": Object {
                "aliases": Array [
                  "Agent Orange",
                  "Braincell",
                  "C-Blast",
                  "Cerebus",
                  "Crushed Insect",
                  "DJ Mystiska K",
                  "Fred",
                  "Kari Pekka",
                  "Magenta",
                  "Mentap",
                  "Mr. James Barth",
                  "Mystic Letter K",
                  "Phunkey Rhythm Doctor",
                  "Rotortype",
                  "Rubberneck",
                  "Shape Changer",
                  "Sir Jeremy Augustus Hutley Of Granith Hall",
                  "Szerementa Programs",
                  "The Mantis (2)",
                  "Vector",
                  "Yakari",
                ],
                "id": 27,
                "name": "Cari Lekebusch",
                "realName": "Kari Pekka Lekebusch",
                "role": "Producer",
                "urls": Array [
                  "http://www.lekebuschmusik.se",
                  "http://blog.carilekebusch.com",
                  "http://de.wikipedia.org/wiki/Cari_Lekebusch",
                  "http://twitter.com/carilekebusch",
                  "http://www.facebook.com/pages/Cari-Lekebusch/36671431790",
                  "http://www.youtube.com/carilekebusch",
                  "http://www.flickr.com/photos/carilekebusch",
                  "http://soundcloud.com/carilekebusch",
                  "http://www.beatport.com/artists/cari+lekebusch",
                  "http://www.residentadvisor.net/dj/carilekebusch",
                  "http://www.alivenotdead.com/CariLekebusch",
                  "http://www.myspace.com/carilekebusch",
                  "http://world.secondlife.com/resident/ce8435c8-f55c-4710-aab4-ac7156078221",
                ],
              },
            },
            Object {
              "node": Object {
                "aliases": Array [
                  "Agent Orange",
                  "Braincell",
                  "C-Blast",
                  "Cerebus",
                  "Crushed Insect",
                  "DJ Mystiska K",
                  "Fred",
                  "Kari Pekka",
                  "Magenta",
                  "Mentap",
                  "Mr. James Barth",
                  "Mystic Letter K",
                  "Phunkey Rhythm Doctor",
                  "Rotortype",
                  "Rubberneck",
                  "Shape Changer",
                  "Sir Jeremy Augustus Hutley Of Granith Hall",
                  "Szerementa Programs",
                  "The Mantis (2)",
                  "Vector",
                  "Yakari",
                ],
                "id": 27,
                "name": "Cari Lekebusch",
                "realName": "Kari Pekka Lekebusch",
                "role": "Recorded By",
                "urls": Array [
                  "http://www.lekebuschmusik.se",
                  "http://blog.carilekebusch.com",
                  "http://de.wikipedia.org/wiki/Cari_Lekebusch",
                  "http://twitter.com/carilekebusch",
                  "http://www.facebook.com/pages/Cari-Lekebusch/36671431790",
                  "http://www.youtube.com/carilekebusch",
                  "http://www.flickr.com/photos/carilekebusch",
                  "http://soundcloud.com/carilekebusch",
                  "http://www.beatport.com/artists/cari+lekebusch",
                  "http://www.residentadvisor.net/dj/carilekebusch",
                  "http://www.alivenotdead.com/CariLekebusch",
                  "http://www.myspace.com/carilekebusch",
                  "http://world.secondlife.com/resident/ce8435c8-f55c-4710-aab4-ac7156078221",
                ],
              },
            },
            Object {
              "node": Object {
                "aliases": Array [
                  "Agent Orange",
                  "Braincell",
                  "C-Blast",
                  "Cerebus",
                  "Crushed Insect",
                  "DJ Mystiska K",
                  "Fred",
                  "Kari Pekka",
                  "Magenta",
                  "Mentap",
                  "Mr. James Barth",
                  "Mystic Letter K",
                  "Phunkey Rhythm Doctor",
                  "Rotortype",
                  "Rubberneck",
                  "Shape Changer",
                  "Sir Jeremy Augustus Hutley Of Granith Hall",
                  "Szerementa Programs",
                  "The Mantis (2)",
                  "Vector",
                  "Yakari",
                ],
                "id": 27,
                "name": "Cari Lekebusch",
                "realName": "Kari Pekka Lekebusch",
                "role": "Written-By",
                "urls": Array [
                  "http://www.lekebuschmusik.se",
                  "http://blog.carilekebusch.com",
                  "http://de.wikipedia.org/wiki/Cari_Lekebusch",
                  "http://twitter.com/carilekebusch",
                  "http://www.facebook.com/pages/Cari-Lekebusch/36671431790",
                  "http://www.youtube.com/carilekebusch",
                  "http://www.flickr.com/photos/carilekebusch",
                  "http://soundcloud.com/carilekebusch",
                  "http://www.beatport.com/artists/cari+lekebusch",
                  "http://www.residentadvisor.net/dj/carilekebusch",
                  "http://www.alivenotdead.com/CariLekebusch",
                  "http://www.myspace.com/carilekebusch",
                  "http://world.secondlife.com/resident/ce8435c8-f55c-4710-aab4-ac7156078221",
                ],
              },
            },
          ],
        },
        "id": 2,
      },
    },
  },
}
`;
